<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>50</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>382</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rocket Chat</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
           <property name="buddy">
            <cstring>UN_Name_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UN_Name_Line">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Login</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RC_Login_Line"/>
         </item>
         <item>
          <widget class="QCheckBox" name="RC_Login_ComboBxo">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password</string>
           </property>
           <property name="buddy">
            <cstring>RC_Pass_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RC_Pass_Line"/>
         </item>
         <item>
          <widget class="QCheckBox" name="RC_Password_comboBox">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Select group</string>
           </property>
           <property name="buddy">
            <cstring>RC_Group_comboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="RC_Group_comboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RC_Add_Button">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RC_Delete_Button">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="RC_Groups_List"/>
       </item>
       <item>
        <widget class="QPushButton" name="RC_Create_Button">
         <property name="text">
          <string>Create Rocket-Chat account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RC_Status_Label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>21</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Overall_Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Overall status</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>12</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remote Desktop</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Domain:</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="RD_Domain_ComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Login:</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RD_Login_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RD_Password_Line"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>E-Mail</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Email</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="EM_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Password</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="EM_Pass_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="EM_Create_Button">
         <property name="text">
          <string>Create E-Mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Email_Status">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Line 24</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Sip extension</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SIP_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>RC_Login_Line</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SIP_Pass_Line"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Create_Excel_File">
       <property name="text">
        <string>Create Excel Card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Excel_Status">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>RC_Login_Line</tabstop>
  <tabstop>RC_Pass_Line</tabstop>
  <tabstop>RC_Password_comboBox</tabstop>
  <tabstop>RC_Login_ComboBxo</tabstop>
  <tabstop>UN_Name_Line</tabstop>
  <tabstop>RC_Create_Button</tabstop>
  <tabstop>RC_Group_comboBox</tabstop>
  <tabstop>RC_Delete_Button</tabstop>
  <tabstop>RC_Groups_List</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
